src/v3.rs

```diff
@@ -126,7 +133,22 @@ impl Sender {
             HeaderValue::from_static("application/json"),
         );
         headers.insert(header::USER_AGENT, HeaderValue::from_static("sendgrid-rs"));
+        Ok(headers)
+    }
 
+    #[cfg(feature = "async")]
```

Comment:
Can you please document this function since it's public?

Reply:
I added documentation and put links to tokio documentation and future documentation.